tests/integrationv2/providers.py

```diff
@@ -248,6 +275,58 @@ def _cipher_to_cmdline(self, protocol, cipher):
 
         return cmdline + ciphers
 
+    @classmethod
+    def supports_max_frag(cls):
+        if '1.0.2' in OpenSSL.get_version():
+            return False
+
+        return True
+
+    @classmethod
+    def get_version(cls):
+        return cls._version
+
+    @classmethod
+    def supports_protocol(cls, protocol, with_cert=None):
```

Comment:
What is the function intended to do? If it is checking for TLS1.3 supported or not, then can we change the function name to `supports_tls13`?

Reply:
Since we pass the protocol in, it acts as a more generic check than just tls13.